tools/supremo.cwl

```diff
@@ -0,0 +1,177 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: supremo
+doc: |
+  Pipeline for generating mutated sequences for input into predictive models
+  and for scoring variants for disruption to genome folding.
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/supremo:1.0.0'
+baseCommand: [">&2", "python", "/opt/SuPreMo/scripts/SuPreMo.py"]
```

Comment:
Ah, you can lead with `>&2`? I forgot about that

Reply:
Yep. Helps clear things up sometimes.